x-pack/plugins/ml/public/application/explorer/_explorer.scss

```diff
@@ -219,9 +219,12 @@
     height: 30px;
```

Comment:
Wonder if we can change this to `$euiSizeXL` (32px) instead?

Reply:
Changed in https://github.com/elastic/kibana/pull/70063/commits/081979542bae2b276991e2ccdfa8a36ab5573ac2